torchao/quantization/qat/__init__.py

```diff
@@ -7,11 +7,13 @@
 from .linear import (
     Int4WeightOnlyQATQuantizer,
     Int8DynActInt4WeightQATQuantizer,
+    SelfCompressionQATQuantizer
 )
 
 __all__ = [
     "ComposableQATQuantizer",
     "Int4WeightOnlyQATQuantizer",
     "Int4WeightOnlyEmbeddingQATQuantizer",
     "Int8DynActInt4WeightQATQuantizer",
+    "SelfCompressionQATQuantizer"
```

Comment:
This feels like a pretty experimental feature. I would exclude it from the top level QAT namespace